docs/classes_1_1deusto_1_1spq_1_1client_1_1gui_1_1views_1_1guest_1_1_hotel_guest_view.html

```diff
@@ -258,12 +258,12 @@ <h2 class="memtitle"><span class="permalink"><a href="#a337f1803e8646e09c876f3c8
       </table>
 </div><div class="memdoc">
 
-<p class="definition">Definition at line <a class="el" href="_hotel_guest_view_8java_source.html#l00032">32</a> of file <a class="el" href="_hotel_guest_view_8java_source.html">HotelGuestView.java</a>.</p>
+<p class="definition">Definition at line <a class="el" href="_hotel_guest_view_8java_source.html#l00031">31</a> of file <a class="el" href="_hotel_guest_view_8java_source.html">HotelGuestView.java</a>.</p>
 
 </div>
 </div>
 <hr/>The documentation for this class was generated from the following file:<ul>
-<li>src/main/java/es/deusto/spq/client/gui/views/guest/<a class="el" href="_hotel_guest_view_8java_source.html">HotelGuestView.java</a></li>
+<li>/Users/Vojta/Documents/CZU/Erasmus/Deusto/SoftwareProcessandQuality/HotelManagement/src/main/java/es/deusto/spq/client/gui/views/guest/<a class="el" href="_hotel_guest_view_8java_source.html">HotelGuestView.java</a></li>
```

Comment:
It started showing your computer's paths

Reply:
Yes, I will commit it again with correct paths